websqrl/pages/index.tsx

```diff
@@ -0,0 +1,12 @@
+export default function IndexPage() {
```

Comment:
the 404 page when visiting `localhost:3000` was a bit confusing to me until i remembered that we split the old index into wikipedia and twitter demo pages